etc/healthchecker/refresh.py

```diff
@@ -0,0 +1,312 @@
+#!/usr/bin/python2
```

Comment:
I'd like to see the tools can support py3 by default. Since the components in openlab are mainly running on py3.

Reply:
Yeah, all of them support py3